shoop/core/templatetags/shoop_common.py

```diff
@@ -23,6 +24,14 @@
 from shoop.utils.serialization import ExtendedJSONEncoder
 
 
+@library.global_function
+def get_localized_language_name(code, name):
+    if code == name:
+        return get_language_info(code.lower())["name_local"]
```

Comment:
I don't think ordering should need to be touched, since `LANGUAGES` is ordered already.